### PR TITLE
Update AWS integration doc with permissions info

### PR DESCRIPTION
**Pull request by jyee**
https://trello.com/c/pUOcjg3n/1071-aws-add-explanations-for-why-we-need-each-permission-in-aws-docs

### CODE REVIEW

File: content/integrations/aws.md [hunk 1]
```diff
@@ -127,25 +127,122 @@ Note: The GovCloud and China regions do not currently support IAM role delegatio
...
+
+* `ecs:ListClusters`: List available clusters.
+* `ecs:ListContainerInstances`: List instances of a cluster.
+* `ecs:DescribeContainerInstances`: Describe instances to add metrics on resources and tasks running, adds cluster tag to ec2 instances.
+
+## Elasticache
+
+* `elasticache:DescribeCacheClusters`: List and describe Cache clusters, to add tags and additional metrics.
+* `elasticache:ListTagsForResource`: List custom tags of a cluster, to add custom tags.
+* `elasticache:DescribeEvents`: Add events avout snapshots and maintenances.
```
Comment (jyee) [inline content/integrations/aws.md]: Typo here "avout"

**Comment by technovangelist** [comment]
So I added another commit to this. I reread the doc and it feels ok. It states that the metrics are coming from cloudwatch and the other permissions allow us to get more context and tags. But I added a link in each section to the aws docs about those IAM perms and I think that completes what is needed.

File: content/integrations/aws.md [hunk 2]
```diff
@@ -52,15 +52,15 @@ There are a number of other AWS services that are also available in Datadog but
...
 {:.table}
 
-
 # Installation
 
-Setting up the Datadog integration with Amazon Web Services requires configuring role delegation using AWS IAM. To get a better understanding of role delegation, refer to the [AWS IAM Best Practices guide](http://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#delegate-using-roles).
+Setting up the Datadog integration with Amazon Web Services requires configuring role delegation using AWS IAM. To get a better 
+understanding of role delegation, refer to the [AWS IAM Best Practices guide](http://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#delegate-using-roles).
 
 Note: The GovCloud and China regions do not currently support IAM role delegation. If you are deploying in these regions please contact support to help set up your AWS integration via API keys.
```
Comment (irabinovitch) [inline content/integrations/aws.md]: China and GovCloud are now supported in the UI again.

**Comment by irabinovitch** [comment]
Going to merge, but lets follow up and add back GovCloud and China.

**Pull request merged**